data.json

```diff
@@ -1855,6 +1855,16 @@
                 "Typescript"
             ],
             "description": "Realtime Web Apps and Dashboards framework for Python and R. Suited (not only) for AI audience."
+        },
+          {
+            "name": "Node Efficientnet",
+            "link": "https://github.com/ntedgi/node-efficientnet",
+            "label": "good first issue",
+            "technologies": [
+                "JavaScript",
+                "Image Recognition"
```

Comment:
Hi, could you please remove this line from `technologies` list? It will otherwise create a whole new section in the document with 1 link in it.